### DISCUSSION TITLE
Adding new agent with no password?

### DISCUSSION THREAD

**Discussion by JakubSiewniak** [chatwoot/chatwoot#3294] [Feature Requests]
Hi.
When I add a new agent (regardless if its through api or website) an email is sent with confirmation link. When said link is clicked, it will verify user but it doesnt prompt the user to set their first password. When a user then logs out, forgotten password has to be used as there is no known password.

Is it possible to force user to set their first password on clicking the activation link?

**Comment by sojan-official** [comment]
we are working on this: https://github.com/chatwoot/chatwoot/issues/3052

**Reply by JakubSiewniak** [reply]
Great. Thank you.
